pytests/stubs/subclassing.pyi

```diff
@@ -0,0 +1,11 @@
+class SubDict(dict):
+    def __new__(cls, /) -> None: ...
+    def __str__(self, /) -> str: ...
+
+class Subclass(Subclassable):
+    def __new__(cls, /) -> None: ...
+    def __str__(self, /) -> str: ...
+
+class Subclassable:
+    def __new__(cls, /) -> None: ...
+    def __str__(self, /) -> str: ...
```

Comment:
I guess technically this isn't valid Python, because `Subclassable` is not yet a real name when line `Subclass` is instantiated.

Should we try to ensure subclasses are after bases?

Reply:
Happy to defer based on that reasoning 👍